### DISCUSSION TITLE
按照沐的推荐论文整理了一个网站收录

### DISCUSSION THREAD

**Discussion by shartoo** [mli/paper-reading#232] [Show and tell]
两年前我在研究StyleGAN的时候找数据和论文找得头大，后来发现太多的优质的博客。于是整了个网站 https://www.webhub123.com/ ，想保存某种热点课题相关的全部网站，希望能帮助我们更好的管理和整合这些博客和论文网站。目前的浏览器收藏夹太简陋了点，只能在文件夹里收藏，然后就再也不会打开。
这里我以 整理的沐神的 [**视频理解** 11 篇经典论文和解读博客以及代码(点击访问)](https://www.webhub123.com/#/home/detail?projectHashid=44498139&ownerUserid=27786724) 为示例，整理了全部相关的论文、相关解读博客以及对应的Github代码，每个网址以一个卡片显示，卡片上有简单的描述、内容预览图、标签等，把它们全部放在一个收录下面，如下图。这样来管理这个方向的研究资料，一目了然。如果要查看具体内容，点击卡片上的图片即可访问
![4744](https://user-images.githubusercontent.com/5716282/218474944-ce227e17-f5c7-4226-b8cf-fc807ee2b9c3.jpg)

我们可能会收藏上万个网址，为了方便管理这些，目前是以层次目录的方式来管理，并加入搜索查询。
![5109](https://user-images.githubusercontent.com/5716282/218475671-19592326-bf9c-4c21-8b1d-be254768f807.jpg)

跟Github一样，我们可以浏览别人分享的一些好的网址收录，这里不仅仅是CV方向的，还包括娱乐、工具、其他方向研究等。看到好的收录，还能一键fork复制到自己的收藏夹中

![335](https://user-images.githubusercontent.com/5716282/218496351-c7f20b05-ae2a-4d53-9986-345383b73432.jpg)

目前网站仍然在逐步扩充内容和不断迭代中，欢迎各位提宝贵的意见和分享自己的一些好网站、博客等。网站所有功能是全免费的，不用担心变相收费问题。

**Comment by geekchen007** [comment]
cool!

**Comment by sharonsunpeng** [comment]
Great resources!
Thanks for sharing!



On Tue, Feb 14, 2023 at 2:53 AM xia ***@***.***> wrote:

> 两年前我在研究StyleGAN的时候找数据和论文找得头大，后来发现太多的优质的博客。于是整了个网站
> https://www.webhub123.com/
> ，想保存某种热点课题相关的全部网站，希望能帮助我们更好的管理和整合这些博客和论文网站。目前的浏览器收藏夹太简陋了点，只能在文件夹里收藏，然后就再也不会打开。
> 这里我以 整理的沐神的 *视频理解* 11
> 篇经典论文为示例，整理了全部相关的论文、相关解读博客以及对应的Github代码，每个网址以一个卡片显示，卡片上有简单的描述、内容预览图、标签等，把它们全部放在一个收录下面，如下图。这样来管理这个方向的研究资料，一目了然。如果要查看具体内容，点击卡片上的图片即可访问
> [image: 4744]
> <https://user-images.githubusercontent.com/5716282/218474944-ce227e17-f5c7-4226-b8cf-fc807ee2b9c3.jpg>
>
> 我们可能会收藏上万个网址，为了方便管理这些，目前是以层次目录的方式来管理，并加入搜索查询。
> [image: 5109]
> <https://user-images.githubusercontent.com/5716282/218475671-19592326-bf9c-4c21-8b1d-be254768f807.jpg>
>
> 目前网站仍然在逐步扩充内容和不断迭代中，欢迎各位提宝贵的意见。网站所有功能是全免费的，不用担心变相收费问题。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mli/paper-reading/discussions/232>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIWUESCHQTBMDSZSS6YLX7DWXI4GJANCNFSM6AAAAAAU2JLJ6M>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Peng SUN,
Ph.D. Candidate in Economics
MA in Economics, BA in English
New Zealand Licensed Immigration Adviser (LIA No. 201700012)
Australia National Accreditation Authority for Translators and Interpreter
Accredited Professional Translator （NAATI No. 56400, from English to
Chinese, accredited in 2005 ）

Mobile: +64-22 5495425

WeChat:maodoumama08
